src/test/scala/cognite/spark/v1/FunctionsUDFTest.scala

```diff
@@ -0,0 +1,47 @@
+package cognite.spark.v1
+
+import cognite.spark.v1.udf.CogniteUdfs
+
+import cats.effect.IO
+import cognite.spark.v1.Constants.{DefaultBaseUrl, DefaultMaxRetries}
+import com.cognite.sdk.scala.v1.GenericClient
+import com.softwaremill.sttp.SttpBackend
+import org.apache.spark.sql.{Row}
+import org.scalatest.{FlatSpec, Inspectors, Matchers, ParallelTestExecution}
+
+
+class FunctionsUDFTest extends FlatSpec with Matchers with ParallelTestExecution with SparkTest with Inspectors {
+
+  it should "read assets with functionUDF" taggedAs ReadTest in {
+    implicit val backend: SttpBackend[IO, Nothing] = CdpConnector.retryingSttpBackend(DefaultMaxRetries)
+    val df = spark.read
+      .format("cognite.spark.v1")
+      .option("apiKey", readApiKey)
+      .option("type", "assets")
+      .option("limitPerPartition", "1")
+      .option("partitions", "1")
```

Comment:
Maybe add a `.option("limit", "50")` or something to this so we don't end up making too many (slow) calls.